### PR TITLE
Add total memory and OS CPU core count to startup telemetry

### PR DESCRIPTION
**Pull request by deekerno**
## Summary
Address IRO-2362 and IRO-2363. Adds stats for total memory and OS CPU core count.

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/telemetry/telemetry.ts [hunk 1]
```diff
@@ -281,11 +281,24 @@ export class Telemetry {
...
+    if (this.metrics) {
+      this.submit({
+        measurement: 'node_started',
+        fields: [
+          { name: 'cpu_cores', type: 'integer', value: this.metrics.cpuCores },
+          { name: 'total_mem', type: 'integer', value: this.metrics.memTotal },
+        ],
+        timestamp,
+      })
+    }
```
Comment (NullSoldier) [inline ironfish/src/telemetry/telemetry.ts:301]: Can we use the existing telemetry above instead of duplicating it?

Comment (deekerno) [reply ironfish/src/telemetry/telemetry.ts]: Fixed.

File: ironfish/src/telemetry/telemetry.ts [hunk 2]
```diff
@@ -281,9 +281,18 @@ export class Telemetry {
   }
 
   submitNodeStarted(): void {
+    const fields: Field[] = [{ name: 'online', type: 'boolean', value: true }]
+
+    if (this.metrics) {
+      fields.concat([
```
Comment (NullSoldier) [inline ironfish/src/telemetry/telemetry.ts:287]: You should double check what this method is doing to see if it does what you want.

File: ironfish/src/telemetry/telemetry.ts [hunk 3]
```diff
@@ -281,9 +281,18 @@ export class Telemetry {
   }
 
   submitNodeStarted(): void {
+    let fields: Field[] = [{ name: 'online', type: 'boolean', value: true }]
+
+    if (this.metrics) {
+      fields = fields.concat([
+        { name: 'cpu_cores', type: 'integer', value: this.metrics.cpuCores },
+        { name: 'total_mem', type: 'integer', value: this.metrics.memTotal },
```
Comment (rohanjadvani) [inline ironfish/src/telemetry/telemetry.ts:289]: nit: Kind of weird it's `total_mem` and the metrics field is `memTotal`. What do you think about `memory_total` to be explicit / consistent?

**Review by rohanjadvani — COMMENTED** [review]
What is the testing plan for this? Can we fill that out or add a screenshot to show this works as intended?

File: ironfish/src/telemetry/telemetry.ts (hunk 3 above)
Comment (NullSoldier) [reply ironfish/src/telemetry/telemetry.ts:289]: Yes please, I agree with Rohan.

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**